
# x: 0.376323
# y: 0.153044
# z: 0.350966
# rot_x: 0.00000333098
# rot_y: 1.0
# rot_z: -0.0000275198
# rot_w: 0.000786376



x: 0.02556177
y: -0.241942
z: 1.03292
rot_x: 0.369633
rot_y: 0.103171
rot_z: 0.643569
rot_w: 0.662229
